zephyr/CMakeLists.txt

```diff
@@ -15,40 +15,14 @@ set(
     "Specify external mbedtls library"
 )
 
-if(CONFIG_OPENTHREAD_COMMISSIONER)
-  set(OT_COMMISSIONER ON CACHE BOOL "Enable Commissioner")
-endif()
-
-if(CONFIG_OPENTHREAD_JAM_DETECTION)
-  set(OT_JAM_DETECTION ON CACHE BOOL "Enable Jam Detection")
-endif()
-
-if(CONFIG_OPENTHREAD_JOINER)
-  set(OT_JOINER ON CACHE BOOL "Enable Joiner")
-endif()
-
-if(CONFIG_OPENTHREAD_SLAAC)
-  set(OT_SLAAC ON CACHE BOOL "Enable SLAAC")
-endif()
-
-if(CONFIG_OPENTHREAD_DHCP6_CLIENT)
-  set(OT_DHCP6_CLIENT ON CACHE BOOL "Enable DHCPv6 Client")
-endif()
-
-if(CONFIG_OPENTHREAD_DHCP6_SERVER)
-  set(OT_DHCP6_SERVER ON CACHE BOOL "Enable DHCPv6 Server")
-endif()
-
-if(CONFIG_OPENTHREAD_DIAG)
-  set(OT_DIAGNOSTIC ON CACHE BOOL "Enable Diagnostics")
-endif()
+# Options imported from common-switches.mk
```

Comment:
I think we shall rather mention `options.cmake` here, as we base on it. It should be up to OpenThread project to keep them in sync with `common-switches.mk`.